src/components/choose-locale/change-locale-online.component.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { postUserPropertiesOnline } from '../../offline/user-properties.resource';
```

Comment:
Sure? Why use it in an `offline` folder if it contains both.

Reply:
That name really isn't good. I'd like to see both fns in the same file though since they are so similar.
I think in this specific case, I can just move it back to `/components/choose-locale/change-locale.esource.tsx`.

Reply:
Would you keep `offline/storage.ts` though?

Reply:
Not sure if we have any naming conventions for this yet..

Reply:
Why don't we have `change-local-online.resource.tsx` and `change-local-offline.resource.tsx`?

The `offline/storage.ts` we can keep, but I'd recommend having `offline.ts` or `offline-storage.ts` instead. A folder with a single file is just overhead.

Reply:
> A folder with a single file is just overhead.

True - I'll go with `src/offline.ts` then which contains: 
* `class PrimaryNavigationDb`
* `type UserPropertiesChange`
* function syncUserPropertiesChanges`

One argument against splitting up the resources file into `change-local-online.resource.tsx` and `change-local-offline.resource.tsx` is that are also shared resources in there (the `type PostUserProperties`) which don't really fit into either online or offline. I guess we could have a third file `change-locale.resource.tsx` which contains them, but wouldn't that also be quite the overhead (we'd end up with 3 files then with a max. of ~10 lines each)?